APMrover2/Parameters.h

```diff
@@ -38,6 +38,8 @@ class Parameters {
         k_param_pivot_turn_angle,
         k_param_rc_13_old,  // unused
         k_param_rc_14_old,  // unused
+        k_param_osd,
```

Comment:
this would set k_param_osd to 20, which is already used

Reply:
I will try to fix it later, just back from office ...